### PR TITLE
Speed up cohorts and use antd

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- Don't show the last event of each user, bringing the speed of the /people pages from ~7 seconds to 0.5
- Redo table etc in antd

Notes:
- When QA-ing make sure to also check clicking on /trends in graph

## Checklist
- [x] All querysets/queries filter by Team (if applicable)
- [x] Backend tests (if applicable)

### CODE REVIEW

File: frontend/src/scenes/users/PeopleTable.js [hunk 1]
```diff
@@ -1,107 +1,75 @@
-import React, { useState } from 'react'
-import { Loading } from 'lib/utils'
+import React, { useEffect, useState } from 'react'
+import { Button, Table } from 'antd'
+import api from 'lib/api'
 import { Link } from 'react-router-dom'
-import moment from 'moment'
+import { FilterLink } from '../../lib/components/FilterLink'
```
Comment (EDsCODE) [inline frontend/src/scenes/users/PeopleTable.js:5]: unneeded now

File: frontend/src/scenes/users/PeopleTable.js [hunk 2]
```diff
@@ -1,107 +1,75 @@
...
+            {Object.keys(properties)
+                .sort()
+                .map(key => (
+                    <div
+                        style={{
+                            flex: '0 1',
+                        }}
+                        key={key}
+                    >
+                        <strong>{key}:</strong> {properties[key]}
```
Comment (EDsCODE) [inline frontend/src/scenes/users/PeopleTable.js:22]: stringify properties[key] if the value is an object

File: frontend/src/scenes/users/PeopleTable.js [hunk 3]
```diff
@@ -1,107 +1,75 @@
...
+                <Link to={'/person/' + encodeURIComponent(person.distinct_ids[0])} className="ph-no-capture">
+                    {person.name}
+                </Link>
+            ),
+        },
+        actions && {
+            title: 'Actions',
+            render: person => (
+                <Button
+                    className="float-right"
```
Comment (EDsCODE) [inline frontend/src/scenes/users/PeopleTable.js:45]: if you remove this line, the 'actions' label at the top of the column will be aligned with the action icon

**Review by EDsCODE — CHANGES_REQUESTED** [review]
Tested with the csv function merged and the export works! Otherwise, found a few bugs and suggested fixes

**Comment by timgl** [comment]
Thanks, all fixed now!

**Comment by EDsCODE** [comment]
Adjustments work! One more thing I found that may seems to have surfaced is now when deleting a user it doesn't auto refresh as it did previously

**Comment by timgl** [comment]
Sorry, fixed!

**Review by EDsCODE — APPROVED** [review]

**Pull request merged**